### ISSUE TITLE
Can't click to select nodes when Show Stems Pro is active

### ISSUE THREAD

**Issue by drawcard** [Mark2Mark/show-stems-pro-plugin#8]
SSP version: 3.1.9
Glyphs: 3.0.5

I can't seem to make any single or drag-selections to nodes, when this plugin is active. Video attached:
https://cln.sh/odlS5juvYkm6AP6DhVZv

**Comment by Mark2Mark** [comment]
Are you holding any keys down?
Works for me:

https://user-images.githubusercontent.com/5824595/155973927-d6adbb82-e975-4caa-975f-01265beaf783.mov

Do you have "Stick To Cursor (beta)" active? That one is kind of buggy at the moment. working on that …

**Comment by drawcard** [comment]
No, sorry, not holding any keys down. But after disabling Stick to Cursor, everything is working again.
